helm-bibtex.el

```diff
@@ -128,15 +131,11 @@ nil, the window will split below."
 ;; work-around.  See also this bug report:
 ;; http://debbugs.gnu.org/cgi/bugreport.cgi?bug=19395
 (defun helm-bibtex-window-width ()
-  (if (and (not (featurep 'xemacs))
```

Comment:
You are of course right that this doesn't do anything useful.  But I do have a faint memory that it used to do something. Hm ...

Reply:
Yeah, I figured that it used to do something. Hopefully it doesn't break anything. Easy to revert if people come complaining, though!